CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - `Maximized` startup mode not filling the screen properly on GNOME Wayland
 - `OptionAsAlt` with `OnlyLeft`/`OnlyRight` settings not working properly on macOS
 - Default Vi key bindings for `Last`/`First` actions not working on X11/Wayland
+- Fixed applying `colors.transparent_background_colors` for selections
```

Comment:
I wasn't sure if all fixes go into the changelog. Lmk if the fix isn't big enough to have here and I can remove it.